### PR TITLE
[ATTEMPT] - Improving Search UI/Frontend

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
tweaked: colors, css fonts, among other things. 
looking forward to hear your feedback. 
kind regards
T. (Tiago)

### CODE REVIEW

File: HelpSource/scdoc.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 body {
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: "Century Gothic", CenturyGothic, AppleGothic, sans-serif;
```
Comment (gusano) [inline HelpSource/scdoc.css]: @tiagmoraismorgado this font is osx only

File: HelpSource/scdoc.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
     font-size: 10pt;
     color: black;
-    background: white;
+    background: white   ;
```
Comment (gusano) [inline HelpSource/scdoc.css]: @tiagmoraismorgado trailing whitespace

**Comment by tiagmoraismorgado** [comment]
here are some screenshots. don't worry I will find another font

captura de ecra 2017-05-30 as 17 11 16
captura de ecra 2017-05-30 as 17 11 42
captura de ecra 2017-05-30 as 17 12 03
captura de ecra 2017-05-30 as 17 12 38

**Comment by tiagmoraismorgado** [comment]
just replaced the font. with a font compatible with linux (i guess). please feel free to check the new version. here are screenshots.

captura de ecra 2017-05-30 as 17 35 13

**Comment by snappizz** [comment]
i'm really sorry, but i think i can speak for others in saying that this isn't an improvement. you can add these changes to your own installation of SC with the file `Platform.userAppSupportDir +/+ "Help/custom.css"`.

**Comment by gusano** [comment]
@tiagmoraismorgado as @snappizz said, a better place for those changes would be on your local machine. These styles are a bit too "personal" IMHO to be applied to the repo.

**Pull request closed without merging**